api/helpers/game-states/convert-card-to-str.js

```diff
@@ -71,7 +71,7 @@ module.exports = {
 
       return exits.success(cardId + attachments);
     } catch (err) {
-      return exits.error('Error at unpacking cards: ' + err.message);
+      return exits.error(sails.__('game.other.errorUnpackCards'));
```

Comment:
I still believe we want to avoid `sails.__` because sails doesn't know the selected lang or where to find the json; so let's just throw the key name